OpenEdXMobile/src/test/java/org/edx/mobile/view/WebViewActivityTest.java

```diff
@@ -23,10 +23,9 @@
 
 public class WebViewActivityTest extends BaseTestCase {
 
-    // TODO: Convert to Parameterized test once we have a Robolectric parameterized test runner.
-
     /**
      * Testing method for displaying web view dialog
+     * TODO: Convert to Parameterized test once we have a Robolectric parameterized test runner.
```

Comment:
This doesn't need to be part of the javadoc; it should be a separate non-doc comment (not starting with `/**`) on this method.